CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -16,21 +19,26 @@ struct EventCellParameters {
 }
 
 class EventDetailsVC: UIViewController {
+    var event : Event?
 
     @IBOutlet weak var imageView: UIImageView!
     @IBOutlet weak var dateLabel: UILabel!
     @IBOutlet weak var titleLabel: UILabel!
-    @IBOutlet weak var locationLabel: UILabel!
     @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var closeButton: UIButton!
+    @IBOutlet weak var locationButton: UIButton!
     
     @IBOutlet weak var topConstraint: NSLayoutConstraint!
     
     var eventTitle: String?
```

Comment:
Are these variables needed if the EventDetailsVC already contains a reference to the Event?